packages/language-server/test/tooltip-spec.ts

```diff
@@ -0,0 +1,188 @@
+import { expect } from "chai";
+import { getHoverResponse } from "../src/hover-tooltip";
+import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+import {
+  generateModel,
+  expectExists,
+} from "@ui5-language-assistant/test-utils";
+import {
+  TextDocument,
+  Position,
+  Hover,
+  TextDocumentIdentifier,
+  TextDocumentPositionParams,
+} from "vscode-languageserver";
+
+describe.only("the UI5 language assistant Hover Tooltip Service", () => {
+  let ui5SemanticModel: UI5SemanticModel;
+  before(async () => {
+    ui5SemanticModel = await generateModel({ version: "1.74.0" });
+  });
+
+  it("will get hover content UI5 property", () => {
```

Comment:
Add `context` according to ast position (e.g. `context("hover on attribute key")` which will have this test, `context("hover on attribute value")` which will have the enum and namespace in xmlns tests)